Source/WebKit/UIProcess/ProcessThrottler.cpp

```diff
@@ -240,6 +240,13 @@ void ProcessThrottler::didConnectToProcess(ProcessID pid)
     updateThrottleStateNow();
     RELEASE_ASSERT(m_assertion || (m_state == ProcessThrottleState::Suspended && !m_shouldTakeSuspendedAssertion));
 }
+
+void ProcessThrottler::didDisconnectFromProcess()
+{
+    PROCESSTHROTTLER_RELEASE_LOG_WITH_PID("didDisconnectFromProcess:", m_processIdentifier);
+    m_processIdentifier = 0;
+    m_assertion = nullptr;
```

Comment:
We should probably `m_dropSuspendedAssertionTimer.stop()` has well, it case we've started this timer. Otherwise, we'll hit a RELEASE_ASSERT() when the timer fires.

Reply:
Similarly, I'd call `clearPendingRequestToSuspend();` to be safe.